### ISSUE TITLE
app_root not work for update-visibility

### ISSUE THREAD

**Issue by TC500** [man-group/dtale#445]
open describe and change column visibility not work when call dtale.show with app_root parameter
because the js call /dtale/update-visibility/ not by app_root setting

```python
import dtale.app as dtale_app
import pandas as pd
dtale_app.JUPYTER_SERVER_PROXY = True
df = pd.DataFrame({'col1':[1,2], 'col2':[3,4]}
PROXY_ROOT = /usr/xxx/proxy
dtale.show(df,ignore_duplicate=True,app_root=PROXY_ROOT)
``` 
then when i change column visibility, the url is not https://<PROXY_ROOT>/dtale/update-visibility/1

**Comment by aschonfeld** [comment]
@TC500 good catch! I’m currently working on adding internationalization but will hopefully be done by the end of the week and then i’ll fix this. 🙏

**Comment by aschonfeld** [comment]
@TC500 just committed a fix for this and a couple other routes.  Hoping to get a new version released in the next couple days.  Will let you know when it is out.  Once again, great catch.

**Comment by aschonfeld** [comment]
Fixed in v1.37.1

@TC500 feel free to toss your ⭐  on the repo and let me know of any other issues or feature requests 🙏

**Comment by TC500** [comment]
@aschonfeld  good job, it works now
